src/index.ml

```diff
@@ -157,17 +157,17 @@ module Make (K : Key) (V : Value) (IO : IO) = struct
 
   let iter_io ?min ?max f io = iter_io_off ?min ?max (fun _ e -> f e) io
 
-  type window = { buf: bytes; off: int64 }
+  type window = { buf : bytes; off : int64 }
 
   let get_entry io ~window off =
     match window with
     | None ->
-      let buf = Bytes.create entry_size in
-      let _ = IO.read io ~off buf in
-      decode_entry buf 0
+        let buf = Bytes.create entry_size in
```

Comment:
is this using ocamlformat?

Reply:
Yes, the 0.10 version as specified by .ocamlformat...But I get the same formatting for 0.11.0

Reply:
would probably make sense to update `.ocamlformat` to use 0.11.0 as well.